gozo-browser-extension/src/index.css

```diff
@@ -11,7 +11,7 @@ html, body, .app {
 }
 
 .app {
-    background-image: url("https://picsum.photos/1920/1080");
+    /* background-image: url("https://picsum.photos/1920/1080"); */
     display: flex;
```

Comment:
comment can be removed if you don't need it here.

Reply:
done